Sources/Datadog/Logger.swift

```diff
@@ -211,9 +223,16 @@ public class Logger {
             level: level,
             message: message,
             date: date,
-            attributes: LogAttributes(userAttributes: combinedAttributes, internalAttributes: nil),
+            attributes: LogAttributes(
+                userAttributes: combinedAttributes,
+                internalAttributes: rumContextIntegration?.currentRUMContextAttributes
+            ),
             tags: tags
         )
+
+        if level.rawValue >= LogLevel.error.rawValue {
```

Comment:
can we make `enum LogLevel: Comparable { ... }`?

Reply:
The `Comparable` methods are not automatically synthesized, so we'd still have to put the `.rawValue` somewhere. Here it's just in a single place, with no impact on the public interface.